src/raftstore/store/peer.rs

```diff
@@ -192,14 +192,14 @@ pub struct PeerStat {
 pub struct Peer {
     engine: Arc<DB>,
     cfg: Rc<Config>,
-    peer_cache: Rc<RefCell<HashMap<u64, metapb::Peer>>>,
+    peer_cache: RefCell<FlatMap<u64, metapb::Peer>>,
```

Comment:
use RefCell for interior mutability, as get_peer_from_cache may insert new elem( ../blob/master/src/raftstore/store/store.rs#L1305)